data/api/identity/v2_pubkey.yaml

```diff
@@ -74,7 +74,7 @@ paths:
         - in: query
           name: public_key
           required: true
-          description: The unpadded base64-encoded public key to check.
+          description: The unpadded standard or URL-safe base64-encoded public key to check.
```

Comment:
note that sydent does *not* accept url-safe keys here (https://vector.im/_matrix/identity/v2/pubkey/isvalid?public_key=ta8IQ0u1sp44HVpxYi7dFOdS_bfwDjcy4xLFlfY5KOA returns `false`, where it would return `true` for a url-safe encoding of that key), so I think this change is incorrect.

Reply:
I reverted the change, but added a link to the definition of unpadded base64.